### PR TITLE
Lorgans bugs that he told me to fix or something

### PR DESCRIPTION
**Pull request by arctic5**
lorgan is too lazy to make a branch for his bugs so he tells me to make a branch to commit his 1 line changes

### CODE REVIEW

**Comment by lorgan3** [comment]
arctic pls this was supposed to be sneaky

**Comment by lorgan3** [comment]
don't merge this yes there will probably be more

**Comment by Medo42** [comment]
@lorgan3 what about now?

**Comment by lorgan3** [comment]
Parallax doesn't really work on multistage (like cp_dirtbowl) maps yet, but I could do that later I guess.

**Comment by TazeTSchnitzel** [comment]
@lorgan3 Add a check for it and show an error in GB or something.

**Comment by lorgan3** [comment]
I'll see if I can make it tomorrow. It's not a bug or a crash but more like a feature that's missing.

**Comment by arctic5** [comment]
lorgan commit these yourself pls

**Comment by lorgan3** [comment]
Ok I'm done with this PR I suppose. That parallax fix will work in game (it fixes the vertical center to the center of the stage instead of the entire map).

The parallax for multistage maps will still look broken in the builder itself because figuring out each stage is harder there and I was in a hurry.

**Comment by arctic5** [comment]
note: if something doesn't work then its lorgans fault for giving me the wrong info to copy paste and commit because he lazy

**Comment by lorgan3** [comment]
^ i tested this branch and it works

File: Source/gg2/Objects/InGameElements/SpriteObject.events/User Event 1.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ if (scale != 1)
 }
 
 fade = readProperty(other.properties, "fade", BOOL, fade);
-animationSpeed = readProperty(other.properties, "animationspeed", REAL, animationSpeed);
+animationSpeed = readProperty(other.properties, "animationspeed", REAL, animationSpeed) * global.delta_factor;
```
Comment (Medo42) [inline Source/gg2/Objects/InGameElements/SpriteObject.events/User Event 1.xml]: This breaks if someone changes between 60fps and 30fps in-game.

File: Source/gg2/Objects/Map elements/Colliders/MovingPlatform.events/User Event 1.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@ if (scale != 1)
     image_yscale = scale;
 }
 
-animationSpeed = readProperty(other.properties, "animationspeed", REAL, animationSpeed);
+animationSpeed = readProperty(other.properties, "animationspeed", REAL, animationSpeed) * global.delta_factor;
```
Comment (Medo42) [inline Source/gg2/Objects/Map elements/Colliders/MovingPlatform.events/User Event 1.xml]: So does this.

**Comment by Medo42** [comment]
Since this is still an improvement vs. not working correctly at all with 60fps, I'll pull it and open an issue for the remaining fix.

**Pull request merged**